tests/integration-tests/conftest.py

```diff
@@ -711,11 +727,31 @@ def inject_additional_config_settings(  # noqa: C901
                     compute_resource["MaxCount"] = 150
 
     configure_scheduler_plugin(scheduler_plugin_configuration, config_content)
+    if inject_odcr:
+        configure_odcr_for_cluster(config_content, region, odcr_stack_factory)
 
     with open(cluster_config, "w", encoding="utf-8") as conf_file:
         yaml.dump(config_content, conf_file)
 
 
+def configure_odcr_for_cluster(cluster_config, region, odcr_stack_factory):
+    if not odcr_stack_factory:
+        return
+    stack_id = odcr_stack_factory(cluster_config)
```

Comment:
Is there any extra cost associated to this stack?

Reply:
`Capacity Reservations are charged at the equivalent On-Demand rate whether you run instances in reserved capacity or not.'
and:
`Billing starts as soon as the Capacity Reservation is provisioned in your account, and it continues while the Capacity Reservation remains provisioned in your account.`

Basically it costs the same as if you had launched all the instances in the reservation. i.e. you pay the cost of the instance whether you use it or not, so if you reserve 10 and only use 6, you still pay for 10.